### PR TITLE
Update feedstock on release

### LINKED ISSUE #28: conda packages
Could we get conda packages published?  flask-restplus previous had these publised to conda forge:
https://anaconda.org/conda-forge/flask-restplus 

The feedstock is here:
https://github.com/conda-forge/flask-restplus-feedstock

### PR DESCRIPTION
**Pull request by SVilgelm**
Updating flask-restx-feedstock repo with new version by creating a pull request when a new release is published.
Close #28 
Tested on my fork: https://github.com/SVilgelm/flask-restx/runs/433937063?check_suite_focus=true
Here is PR for the version 0.1.1 created by release action: https://github.com/conda-forge/flask-restx-feedstock/pull/7

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -26,3 +26,75 @@ jobs:
...
+        shell: python
+        run: |
+          from http import client
+          from pkg_resources import parse_requirements
+          import re
+          VERSION = "${{ steps.vars.outputs.version }}"
+          with open("feedstock/recipe/meta.yaml") as f:
+              meta = f.read()
+          requirements = []
+          with open("base/requirements/install.pip") as f:
```
Comment (ziirish) [inline .github/workflows/release.yml]: I think we should first setup dependabot which requires the files to have the `.txt` extension. This way we won't forget to update it later.

Comment (SVilgelm) [reply .github/workflows/release.yml]: hmm, good point, but I'm not sure we can add Dependabot soon :)

Comment (SVilgelm) [reply .github/workflows/release.yml]: I'll create a PR for Dependabot, so we can start discussion

Comment (SVilgelm) [reply .github/workflows/release.yml]: @ziirish I want to merge it before 0.2.0. I will change the workflow as soon as we dropped 2.7 and 3.4

**Pull request merged**